recipes/r-tibble/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-tibble
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "1.2"
+
+source:
+  fn: tibble_1.2.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/tibble_1.2.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/tibble/tibble_1.2.tar.gz
+  md5: bdbc3d67aa16860741add6d6ec20ea13
+
+
+  # You can add a hash for the file here, like md5 or sha1
+  # md5: 49448ba4863157652311cc5ea4fea3ea
+  # sha1: 3bcfbee008276084cbb37a2b453963c61176a322
+  # patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: testthat, withr, knitr (>= 1.5.32), rmarkdown, nycflights13, microbenchmark
+requirements:
+  build:
+    - r-base
+    - r-rcpp
+    - r-assertthat
+    - r-lazyeval >=0.1.10
+    - posix               # [win]
+    - {{native}}toolchain # [win]
```

Comment:
This looks strange. Can we remove the windows specific bits as we do not support Windows and probably never will?